src/python/WMCore/WMStats/CherryPyThreads/CleanUpTask.py

```diff
@@ -20,17 +20,7 @@ def setConcurrentTasks(self, config):
         """
         sets the list of functions which runs concurrently
         """
-        self.concurrentTasks = [{'func': self.cleanUpOldRequests, 'duration': (config.DataKeepDays * 24 * 60 * 60)},
-                                {'func': self.cleanUpArchivedRequests, 'duration': config.archivedCleanUpDuration}]
```

Comment:
Don't you want to keep this second line?

Reply:
Actually I didn't remove the second line.
https://github.com/dmwm/WMCore/pull/8836/files#diff-a83e41bb095c827fc02a0bc6f4694b4cR23